### DISCUSSION TITLE
Do I actually need the toshiba RBC-AMT31 remote to be able to control it?

### DISCUSSION THREAD

**Discussion by voipmonitor** [issalig/toshiba_air_cond#49] [Q&A]
I have toshiba daisekai roof units all over my house and I'm curious if I need the controller RBC-AMT31 to be able to actually control this unit just by wiring AB to this project? 

I'm also about to order the board from jlcpcb.com and I was thinking about placing all the pieces on it by them which they offer too but there is no such files for it in the project so I suppose everyone did it themselves?

**Comment by issalig** [comment]
If you refer to the bill of materials, you can get it from the schematics https://github.com/issalig/toshiba_air_cond/blob/master/pcb/airpcb_smd/schematics.pdf  or https://github.com/issalig/toshiba_air_cond/blob/master/pcb/airpcb_smd/airpcb_smd.sch looking for F 0 and F 1 tags

For example: 
$ grep "F [01]" *sch
F 0 "R2" H 1950 1796 50  0000 L CNN
F 1 "10k" H 1950 1705 50  0000 L CNN
F 0 "R3" V 1823 1600 50  0000 C CNN
F 1 "100" V 1914 1600 50  0000 C CNN
F 0 "D2" H 2330 1817 50  0000 C CNN
F 1 "1N4001" H 2330 1726 50  0000 C CNN
F 0 "U1" H 2780 2025 50  0000 C CNN
F 1 "PC817" H 2780 1934 50  0000 C CNN
F 0 "R5" H 3150 1996 50  0000 L CNN
F 1 "1k" H 3150 1905 50  0000 L CNN
F 0 "D1" H 1730 2117 50  0000 C CNN
F 1 "13" H 1730 2026 50  0000 C CNN
F 0 "#PWR03" H 3080 1850 50  0001 C CNN
F 1 "GND" H 3085 1927 50  0000 C CNN
F 0 "U3" H 9755 1220 50  0000 C CNN
F 1 "WeMos_D1_mini" H 9755 1120 50  0000 C CNN
F 0 "#PWR011" H 9755 2870 50  0001 C CNN
F 1 "GND" H 9760 2947 50  0000 C CNN
F 0 "J4" H 5875 947 50  0000 L CNN
F 1 "AB" H 5875 856 50  0000 L CNN
F 0 "J2" H 5185 2742 50  0000 L CNN
F 1 "DHT" H 5185 2651 50  0000 L CNN
F 0 "#PWR05" H 4905 2700 50  0001 C CNN
F 1 "GND" H 4910 2777 50  0000 C CNN
F 0 "R7" V 4398 2750 50  0000 C CNN
F 1 "10k" V 4489 2750 50  0000 C CNN
F 0 "J1" H 4885 3642 50  0000 L CNN
F 1 "SPI" H 4885 3551 50  0000 L CNN
F 0 "#PWR04" H 4605 3400 50  0001 C CNN
F 1 "GND" V 4610 3522 50  0000 R CNN
F 0 "U4" H 9855 6246 50  0000 C CNN
F 1 "ESP-12E" H 9855 6155 50  0000 C CNN
F 0 "J7" H 5835 3742 50  0000 L CNN
F 1 "PWR" H 5835 3651 50  0000 L CNN
F 0 "#PWR07" H 5555 3550 50  0001 C CNN
F 1 "GND" V 5560 3672 50  0000 R CNN
F 0 "#PWR012" H 9855 5715 50  0001 C CNN
F 1 "GND" H 9860 5792 50  0000 C CNN
F 0 "#PWR010" H 7625 3360 50  0001 C CNN
F 1 "VCC" V 7640 3637 50  0000 L CNN
F 0 "#PWR01" H 2050 3850 50  0001 C CNN
F 1 "GND" H 2055 3927 50  0000 C CNN
F 0 "Q1" H 2140 3946 50  0000 L CNN
F 1 "PN2222A" H 2140 3855 50  0000 L CNN
F 0 "#PWR02" H 2600 3650 50  0001 C CNN
F 1 "GND" H 2605 3727 50  0000 C CNN
F 0 "D4" H 3650 3917 50  0000 C CNN
F 1 "1N4001" H 3650 3826 50  0000 C CNN
F 0 "Q2" H 3590 4246 50  0000 L CNN
F 1 "PN2222A" H 3590 4155 50  0000 L CNN
F 0 "R4" H 1980 3504 50  0000 R CNN
F 1 "200" H 1980 3595 50  0000 R CNN
F 0 "D3" V 3454 3930 50  0000 L CNN
F 1 "13" V 3545 3930 50  0000 L CNN
F 0 "U2" H 2900 4125 50  0000 C CNN
F 1 "PC817" H 2900 4034 50  0000 C CNN
F 0 "R6" H 3130 4004 50  0000 R CNN
F 1 "1k" H 3130 4095 50  0000 R CNN
F 0 "R1" V 1393 3900 50  0000 C CNN
F 1 "1k" V 1484 3900 50  0000 C CNN
F 0 "J8" H 5930 2842 50  0000 L CNN
F 1 "WS2812" H 5930 2751 50  0000 L CNN
F 0 "#PWR08" H 5650 2650 50  0001 C CNN
F 1 "GND" V 5655 2772 50  0000 R CNN
F 0 "J11" H 5865 1667 50  0000 L CNN
F 1 "PSU" H 5865 1576 50  0000 L CNN
F 0 "J3" V 5385 1325 50  0000 L CNN
F 1 "DC" V 5285 1280 50  0000 L CNN
F 0 "#PWR06" H 5585 1575 50  0001 C CNN
F 1 "GND" H 5590 1652 50  0000 C CNN
F 0 "R8" V 4528 1575 50  0000 C CNN
F 1 "100" V 4619 1575 50  0000 C CNN
F 0 "D5" H 5035 1792 50  0000 C CNN
F 1 "1N4001" H 5035 1701 50  0000 C CNN
F 0 "J5" H 7183 2472 50  0000 C CNN
F 1 "Conn_01x08" H 7183 2381 50  0000 C CNN
F 0 "J6" H 7873 2482 50  0000 C CNN
F 1 "Conn_01x08" H 7873 2391 50  0000 C CNN
F 0 "#PWR0101" H 8155 1515 50  0001 C CNN
F 1 "GND" V 8160 1637 50  0000 R CNN
F 0 "#PWR0102" H 7630 3605 50  0001 C CNN
F 1 "VCC" V 7645 3882 50  0000 L CNN
F 0 "#PWR0103" H 7615 3655 50  0001 C CNN
F 1 "GND" V 7620 3777 50  0000 R CNN
F 0 "#FLG0101" H 7630 3830 50  0001 C CNN
F 1 "PWR_FLAG" V 7630 3883 50  0000 L CNN
F 0 "#PWR0104" H 7615 3705 50  0001 C CNN
F 1 "GNDPWR" V 7620 3797 50  0000 R CNN
F 0 "#FLG0102" H 7590 4220 50  0001 C CNN
F 1 "PWR_FLAG" V 7590 4273 50  0000 L CNN
F 0 "#FLG0103" H 7590 4320 50  0001 C CNN
F 1 "PWR_FLAG" V 7590 4373 50  0000 L CNN
F 0 "J9" H 5200 2247 50  0000 L CNN
F 1 "5V3V" H 5200 2156 50  0000 L CNN

**Reply by voipmonitor** [reply]
thank you - and do you know if the RBC-AMT31 remote controller is needed to be able to talk to the toshiba? Like just wiring A/B to the ESP board? I think yes?

**Comment by issalig** [comment]
My board mimicks  RBC-AMT31 . You can have both or just my board.
